### ISSUE TITLE
Assets update only once when the files change.

### ISSUE THREAD

**Issue by ladace** [bjorn/tiled#418]
In most cases, I open my map in Tiled, and then edit the image of the tiles using Photoshop. When I save the file and switch back to Tiled, the map will update (sometimes won't). And then I continue to edit the image with Tiled opening, and save the file, however, every time I switch back, the map won't update and I have to restart it.

The assets will only update once after the files change. It is really inconvenient. This is a small bug and I hope you'll fix it.

Tiled 0.9.0 on Mac.

**Comment by bjorn** [comment]
Closing this since I noticed it's a duplicate of issue #44.
